server/routes/auth.ts

```diff
@@ -714,17 +714,66 @@ authRoutes.post('/local', async (req, res, next) => {
   }
 });
 
-authRoutes.post('/logout', (req, res, next) => {
-  req.session?.destroy((err) => {
-    if (err) {
-      return next({
-        status: 500,
-        message: 'Something went wrong.',
-      });
+authRoutes.post('/logout', async (req, res, next) => {
+  try {
+    const userId = req.session?.userId;
+    if (!userId) {
+      return res.status(200).json({ status: 'ok' });
     }
 
-    return res.status(200).json({ status: 'ok' });
-  });
+    const user = await getRepository(User)
+      .createQueryBuilder('user')
+      .addSelect(['user.jellyfinUserId', 'user.jellyfinDeviceId'])
+      .where('user.id = :id', { id: userId })
+      .getOne();
+
+    if (user?.jellyfinUserId && user.jellyfinDeviceId) {
+      try {
+        const settings = getSettings();
+        if (!settings.jellyfin.apiKey) {
+          throw new Error('No Jellyfin API key configured');
+        }
```

Comment:
I'm not sure it's in the scope of this controller to check if there is a Jellyfin server configured. We don't check it elsewhere.
@fallenbagel wdyt?